### PR TITLE
Refs [TASK][25969] Improve comments in campaigns by using rails ajax

### PR DESCRIPTION
**Pull request by hoatb-1022**
https://edu-redmine.sun-asterisk.vn/issues/25969

Screen Shot 2020-07-27 at 10 30 34

### CODE REVIEW

**Comment by hoatb-1022** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@ def full_title page_title, is_admin = false
     page_title.blank? ? base_title : [page_title, base_title].join(" | ")
   end
 
+  def full_admin_title page_title
+    base_title = t "global.app_admin_title"
+    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
+  end
```
Comment (manhnt-0960) [inline app/helpers/application_helper.rb]: method này thừa này e

Comment (hoatb-1022) [reply app/helpers/application_helper.rb]: Ô chắc em fix conflict quên k xoá :(

**Comment by hoatb-1022** [comment]
ready

**Pull request merged**